modules/ROOT/pages/branded-emails.adoc

```diff
@@ -0,0 +1,66 @@
+= Optionally Configure Branded Emails
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Anypoint API Community Manager sends notification emails when an API contract
+is approved or revoked. By default these emails use Anypoint Platform addresses and
+branding. If you wish to configure them to use your own addresses and branding
+instead, use these steps.
+
+. Navigate to *Setup* > *Home* > *Integrations* > *External Data Sources* and ensure *Eligible for External Data Change Capture* is enabled.
+. Set *Polling Interval for External Change Data Capture (seconds)*.
++
+The default is 1800 and the minimum is 300.
+. Navigate to *Setup*, use *Quick Find* to go to *Change Data Capture*, navigate to *Entities* > *Change Manually* > *Available Entities*, and move *AnypointContracts* to the *Selected Entities* list.
+. Navigate to *Setup*, use *Quick Find* to go to *Apex Triggers*, and click the *Developer Console* button.
+. Click *File* > *New* > *Apex Trigger*, set *Name* to `AnypointEmailNotificationsTrigger`, and set *sObject* to `acm_pkg__AnypointContracts__ChangeEvent`.
+. Enter this code block:
++
+----
+    trigger AnypointEmailNotificationsTrigger on acm_pkg__AnypointContracts__ChangeEvent (after insert) {
+        List<acm_pkg__AnypointContracts__ChangeEvent> events = Trigger.new;
+        acm_pkg__.AnypointEmailNotifications.notifyOnContractUpdates(events);
+    }
+----
++
+. Grant the *Organization Administrator* role to the Anypoint user named *Automated Process*.
```

Comment:
This step should be at the end. The anypoint user "automated process" will be created automatically in anypoint only after this new functionality is exercised for the first time (i.e. after all the config is done, and a user requests a contract and it is then approved in anypoint). 

Maybe this should be more like a small section at the end "Complete permissions configuration in anypoint" or something like that, that includes 
1) exercising it for the first time: as the admin, go to the public side of the community, request access to an API with manual approval and approve it in anypoint - this creates the automated process user
2) giving org admin to the new automated user that was created

Reply:
Added. Thank you!